pkg/api/aim/encoding/decoder.go

```diff
@@ -49,7 +49,7 @@ func Decode(data io.Reader) (map[string]interface{}, error) {
 	for {
 		key, err := d.readField()
 		if err != nil {
-			if err := eris.Unwrap(err); err == io.EOF {
+			if err = eris.Unwrap(err); err == io.EOF {
```

Comment:
This feels dangerous to me! Better to use a different name if you want to avoid the linter yelling at you for overshadowing a variable. In this case, I _think_ that `err` will become the unwrapped `err` — whether it's an `io.EOF` or not — and then we may wrap it and return it and then loose some parts of it.

Reply:
yes yes, fixed.